app/src/main/java/com/github/se/wanderpals/ui/screens/trip/finance/FinancePieChart.kt

```diff
@@ -26,13 +26,15 @@ import com.github.se.wanderpals.model.data.Expense
  * @param chartBandWidth The width of the pie chart bands.
  * @param totalValueDisplayIsEnabled boolean value to indicate if the total value of the expenses
  *   has to be displayed at the center of the pie-chart.
+ * @param currencySymbol Symbol of the currency.
  */
 @Composable
 fun FinancePieChart(
     expenses: List<Expense>,
     radiusOuter: Dp = 100.dp,
     chartBandWidth: Dp = 20.dp,
-    totalValueDisplayIsEnabled: Boolean = false
+    totalValueDisplayIsEnabled: Boolean = false,
+    currencySymbol: String = "CHF"
```

Comment:
You should remove the default setting here to stay consistent with the rest of the code since for other arguments in other functions, you didn't set it. Or make it a variable in strings.xml